### DISCUSSION TITLE
Does OpenFeature provides a backend storage for feature-flags or it merely provides and interface that 3rd party solutions can be plugged into?

### DISCUSSION THREAD

**Discussion by deepthyvalsan** [open-feature/community#267] [Q&A] [answered]
Does OpenFeature provides a backend storage for feature-flags or it merely provides and interface that 3rd party solutions can be plugged into?

**Comment by thomaspoignant** [comment, marked as answer]
Hello @deepthyvalsan, 
I just saw your discussion here.

OpenFeature main goal is to be the interface to different solutions, but it still provides a back-end solution called `flagD`.

You have other alternatives also if you want to check, you can start by looking at the ecosystem page in the documentation https://openfeature.dev/ecosystem/.
